examples/common_spaces.jl

```diff
@@ -35,14 +35,15 @@ function make_horizontal_space(
     mesh,
     npoly,
     context::ClimaComms.SingletonCommsContext,
+    HorizontalLayout = DataLayouts.IJFH,
```

Comment:
Could the `horizontal_layout` keyword argument use camel case instead? That would be a little more standard.

Reply:
I did think about this, my main motivation for using TitleCase was that it is a Type, I don't have strong preferences though. @sriharshakandala, what do you think?